src/prefect/engine/result_handlers/pandas_result_handler.py

```diff
@@ -0,0 +1,148 @@
+import pandas as pd
+import os
+import typing
+
+from prefect.engine.result_handlers.result_handler import ResultHandler
+
+
+class PandasResultHandler(ResultHandler):
+    """
+    Hook for storing and retrieving task results in Pandas DataFrames.
+    Task results are written/read via the standard Pandas
+    ``to_[FILETYPE]``/``read_[FILETYPE]`` methods, and additionally the filename
+    can be fully specified at task instantiation.
+
+    Args:
+        - path (str): Filepath to be read from or written to, including file name and extension.
+        - file_type (str): The type of file to write to, e.g. "csv" or "parquet". Must match the name
+            of the appropriate ``to_[FILETYPE]``/``read_[FILETYPE]`` method.
+        - read_kwargs (dict, optional): If present, passed as **kwargs to the ``read_[FILETYPE]`` method.
+        - write_kwargs (dict, optional): If present, passed as **kwargs to the ``to_[FILETYPE]`` method.
+
+    .. note::
```

Comment:
We actually support markdown in our docstrings, so you might change this to:
```suggestion
    **Note:**
```

Reply:
Oh right, my bad. I must be brainwashed by Sphinx 😛